src/classes/Forms.ts

```diff
@@ -787,6 +788,39 @@ export default class Forms extends OneBlinkAPI {
       `/forms/${formId}${overrideLock ? '?overrideLock=true' : ''}`,
     )
   }
+  /**
+   * #### Example
+   *
+   * ```javascript
+   * const formId = 1
+   * forms
+   *   .migrateForm({
+   *      formsAppEnvironmentId: 2,
+   *      sourceFormId: 123,
+   *      targetFormId: 234,
+   *      elements: true,
+   *      submissionEvents: false,
+   *      serverValidation: false,
+   *      externalIdGeneration: false,
+   *      postSubmissionAction: false,
+   *   })
+   *   .then((migratedForm) => {
+   *     // do something with form
+   *   })
+   *   .catch((error) => {
+   *     // Handle error here
+   *   })
+   * ```
+   *
+   * @param formId Id of the form.
+   * @param migrationOptions Migration options
+   */
+  async migrateForm(migrationOptions: FormMigrationOptions): Promise<void> {
+    await super.postRequest<FormMigrationOptions, FormTypes.Form>(
+      `/forms/${migrationOptions.sourceFormId}/migrate`,
+      migrationOptions,
+    )
+  }
```

Comment:
Thoughts on doing checks on whether supplied ids are given as numbers like other some other functions do? 
eg.
```
if (!data || typeof data.id !== 'number') {
      throw new TypeError('Must supply "EmailTemplate.id" as a number')
    }
```
IMO, not really necessary as the api will return an error anyway

Reply:
i started doing that but then looking at the other functions, some do it and some dont - mostly the ones that take an object dont check the values. I figured the API would do it anyways and the user would get feeback from that